doc/sphinxext/gen_rst.py

```diff
@@ -8,19 +8,18 @@
 Files that generate images should start with 'plot'
```

Comment:
Looks like there are still have changes in doc/sphintext that you haven't reverted